sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -41,7 +42,9 @@
     private final Map<String, Protos.TaskInfo.Builder> taskBuilders = new HashMap<>();
     private Protos.ExecutorInfo.Builder executorBuilder;
     private final PodInstance podInstance;
+    // TODO(nickbp): Remove this env storage after October 2017 when it's no longer used as a fallback for dynamic ports
```

Comment:
The env storage is _only_ currently used to autodetect prior values for dynamic ports. Once we've had a couple dcos-commons releases where stored TaskInfos have the port name set, it can be removed in favor of that mechanism instead.